app/controllers/openstax/api/v1/api_controller.rb

```diff
@@ -46,8 +48,12 @@ def session_user?
           !!current_session_user && doorkeeper_token.blank?
         end
 
+        def set_date_header
+          response.date = Time.now unless response.date?
```

Comment:
This is the purpose of this header according to http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.18
Also according to the RFC, it should pretty much always be set (idk why Rails/thin don't do it).